lib/kayvee.ts

```diff
@@ -13,8 +14,8 @@ function replaceErrors(key, value) {
 
 // Converts a map to a string space-delimited key=val pairs
 function format(data) {
-  if (deploy_env) {
-    return JSON.stringify(_.extend({deploy_env}, data), replaceErrors);
+  if (deploy_env || workflow_id) {
+    return JSON.stringify(_.extend({deploy_env, workflow_id}, data), replaceErrors);
```

Comment:
We should only write workflow_id when it is set

Reply:
Changed in 966c876